icefoganalytics/travel-authorization
# Update Current User Vuex Store Pattern Usage

Fixes https://github.com/icefoganalytics/travel-authorization/issues/106

# Context

Update the Vuex store pattern that is being used by the "currentUser" store, so that:
- its called `current/user`
- it's using `ensure` instead of `initialize`
- its using caching i.e. while isLoading await sleep
- it has an `isReady` getter.

# Testing Instructions

1. Run the test suite via `dev test` (or `dev test_api`)
2. Boot the app via `dev up`
3. Log in to the app at http://localhost:8080
4. Run through the My Travel Request and Manager View workflows.
5. Check that My Profile interface works correctly as well.
